### PR TITLE
feat: Detailed Long-Term relationship Timeline

### PR DESCRIPTION
**Pull request by dricazenck**
* Update timeline in home page
image

* Create a detailed timeline for a long-term relationship 

image

### CODE REVIEW

File: timeline.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <h1>Long-Term Relationship</h1>
+
+    <div class="container mt-6 mb-6">
+        <div class="row">
+            <div class="col-md-7 offset-md-3">
+                <ul class="timeline">
+                    <li>
+                        <div class="date">22nd Jan, 2023</div>
+                        <h4>Registration for mentors is open</h4>
+                        <div class="description">Registration for mentors will be open during the complete cicly. Allowing ad-hoc sessions</div>
```
Comment (nora-weisser) [inline timeline.html:15]: Mentors can apply for Long-Term Mentorship Relationships till 10th of March. Once a profile has been approved and published on the website, the mentor will receive a confirmation email.

I think we can keep registration open for long-term mentorship only till 10th of March to be able to match mentors and mentees.

File: timeline.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    </li>
+                    <li>
+                        <div class="date">22nd - 13th of March, 2023</div>
+                        <h4>Registration for mentors is open</h4>
```
Comment (nora-weisser) [inline timeline.html:19]:
```suggestion
                        <h4>Registration for mentees is open</h4>
```

File: timeline.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                        <div class="description">Registration for mentees it closes only for Long-Term relationship, but will open again for ad-hoc sessions</div>
```
Comment (nora-weisser) [inline timeline.html:20]:
```suggestion
                        <div class="description">Mentees will have an opportunity to choose a mentor and apply for Long-Term Mentorship Relationships till 13th of March.</div>
```

File: timeline.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                    </li>
+                    <li>
+                        <div class="date">8th of March, 2023</div>
+                        <h4>Panel Discussion</h4>
+                        <p>Panel Discussion “What makes a good mentor?” check the event page and be ready to collect more informations about to be a mentor and mentee.</p>
```
Comment (nora-weisser) [inline timeline.html:25]:
```suggestion
                        <p>The discussion will cover topics such as communication, active listening, setting clear expectations, providing guidance and support, giving feedback, promoting growth and development, etc. Moreover, panel discussion will examine the role of mentorship, challenges and opportunities that come with mentoring. </p>
```

File: timeline.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+                        <div class="description">Registration for mentors will be open during the complete cicly. Allowing ad-hoc sessions</div>
+                    </li>
+                    <li>
+                        <div class="date">22nd - 13th of March, 2023</div>
+                        <h4>Registration for mentors is open</h4>
+                        <div class="description">Registration for mentees it closes only for Long-Term relationship, but will open again for ad-hoc sessions</div>
+                    </li>
+                    <li>
+                        <div class="date">8th of March, 2023</div>
+                        <h4>Panel Discussion</h4>
```
Comment (nora-weisser) [inline timeline.html:24]:
```suggestion
                        <h4>Panel Discussion “What makes a good mentor?”</h4>
```

File: timeline.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+                        <p>Panel Discussion “What makes a good mentor?” check the event page and be ready to collect more informations about to be a mentor and mentee.</p>
+                    </li>
+                    <li>
+                        <div class="date">13th March - 1st of April, 2023</div>
+                        <h4>Matching process</h4>
+                        <div class="description">Mentors will receive the applications and depending of their availability will accept the mentee</div>
```
Comment (nora-weisser) [inline timeline.html:30]:
```suggestion
                        <div class="description">Once the application period has ended, the Mentorship Programme Team will review the applications. Then list of selected mentees will be sent to the mentors, along with their profiles and information about their goals and expectations for the mentorship. Based on the availability mentors will be able to accept mentees. </div>
```

File: timeline.html [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+                    </li>
+                    <li>
+                        <div class="date">11th of April, 2023</div>
+                        <h4>Introduction emails</h4>
+                        <div class="description">Introduction emails will be sent to mentors and mentees</div>
```
Comment (nora-weisser) [inline timeline.html:35]:
```suggestion
                        <div class="description">Once list with matched mentors and mentees is ready, all participants will receive introduction email with further steps.</div>
```

File: timeline.html [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+                    </li>
+                    <li>
+                        <div class="date">April - November, 2023</div>
+                        <h4>Regular meetings</h4>
+                        <div class="description">Mentors and mentees will schedule regular meetings to discuss progress and share advice and feedback. The frequency of these meetings will depend on the preferences of the mentor and mentee, but they will typically happen every 1-2 weeks</div>
+                    </li>
+                    <li>
+                        <div class="date">July and October, 2023</div>
+                        <h4>Programme check-ins</h4>
+                        <div class="description">Two times during the cycle Mentors and mentees will be invited to a mentorship check-in meeting to assess progress and collect feedback</div>
```
Comment (nora-weisser) [inline timeline.html:55]:
```suggestion
                        <div class="description">Two times during the cycle mentors and mentees will be invited to a mentorship check-in meeting to assess progress and collect feedback</div>
```

File: index.html [hunk 9]
```diff
@@ -89,11 +103,15 @@ <h5 class="card-title">Mentoring Starts</h5>
                     </div>
                     <div class="card-body">
                         <h5 class="card-title">The end of the Cycle</h5>
```
Comment (nora-weisser) [inline index.html:105]:
```suggestion
                        <h5 class="card-title">The End of the Cycle</h5>
```

File: timeline.html (hunk 1 above)
Comment (dricazenck) [inline timeline.html:15]:
```suggestion
                        <div class="description">Mentors can apply for Long-Term Mentorship Relationships till 10th of March. Once a profile has been approved and published on the website, the mentor will receive a confirmation email.</div>
```

File: timeline.html [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+---
+<div class="timeline-page container">
+    <h1>Long-Term Relationship</h1>
+
+    <div class="container mt-6 mb-6">
+        <div class="row">
+            <div class="col-md-7 offset-md-3">
+                <ul class="timeline">
+                    <li>
+                        <div class="date">22nd Jan, 2023</div>
```
Comment (dricazenck) [inline timeline.html:13]:
```suggestion
                        <div class="date">22nd Jan, 2023</div>
```

File: timeline.html [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+                    </li>
+                    <li>
+                        <div class="date">November, 2023</div>
+                        <h4>End of the Programme</h4>
+                        <div class="description">The mentor and mentee will agree in a final meeting to conclude the </div>
+                            Mentorship Programme Team will conduct the final meeting for all mentors and mentees to wrap up the mentorship programme and discuss outcomes. </p>
+                    </li>
+                    <li>
+                        <div class="date">November and December, 2023</div>
+                        <h4>Program evaluation and feedback</h4>
```
Comment (dricazenck) [inline timeline.html:65]:
```suggestion
                        <h4>Programme evaluation and feedback</h4>
```

File: timeline.html [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+        <div class="row">
+            <div class="col-md-7 offset-md-3">
+                <ul class="timeline">
+                    <li>
+                        <div class="date">22nd Jan - 10th of March, 2023</div>
+                        <h4>Registration for mentors is open</h4>
+                        <div class="description">Mentors can apply for Long-Term Mentorship Relationships till 10th of March. Once a profile has been approved and published on the website, the mentor will receive a confirmation email.</div>
+                    </li>
+                    <li>
+                        <div class="date">22nd - 13th of March, 2023</div>
```
Comment (nora-weisser) [inline timeline.html:18]:
```suggestion
                        <div class="date">22nd of February - 13th of March, 2023</div>
```

**Review by nora-weisser — APPROVED** [review]

**Pull request merged**